js/signin.js

```diff
@@ -0,0 +1,114 @@
+const QUERY_SELECTOR = function (select) {
+  return document.querySelector(select);
+};
+
+const $emailInput = QUERY_SELECTOR('#emailInput');
+const $emptyEmailError = QUERY_SELECTOR('#emptyEmailError');
+const $wrongEmailError = QUERY_SELECTOR('#wrongEmailError');
+
+$emailInput.addEventListener('focusout', function (e) {
```

Comment:
focusout과 blur의 차이에 대해 알아봐도 좋을 것 같아요!

Reply:
이벤트 핸들러 함수를 작성할 때, 내부에서 이벤트 객체(e)를 사용하지 않는다면 비워놓는게 좋습니다!
```suggestion
$emailInput.addEventListener('focusout', function () {
```